### PR TITLE
Removing unused .DLL

### PR DESCRIPTION
**Pull request by Xphalnos**
I finished the cleanup of CMakeLists.txt by deleting what I had forgotten to delete.
I modified the Windows-Qt workflow to not download unnecessary .DLL files.
This reduces the size of the artifacts by almost 10MB.
This also reduces compilation time.
I took the opportunity to update the submodules.
Tested and working!

Before:
image

After:
Capture d'écran 2024-07-12 133613

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -62,17 +53,17 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 find_package(Boost 1.84.0 CONFIG)
 find_package(cryptopp 8.9.0 MODULE)
-find_package(fmt 10.2.1 CONFIG)
-find_package(glslang 14.2.0 CONFIG)
+find_package(fmt 11.0.1 CONFIG)
```
Comment (raphaelthegreat) [inline CMakeLists.txt]: I would avoid needlessly updating packages all the time. Linux distros don't always support latest versions

**Comment by Xphalnos** [comment]
Done

**Comment by raphaelthegreat** [comment]
Can you also remove upgrades to other packages? I feel would be better to do upgrades in separate PRs

File: CMakeLists.txt [hunk 2]
```diff
@@ -590,7 +581,5 @@ add_dependencies(shadps4 host_shaders)
 target_include_directories(shadps4 PRIVATE ${HOST_SHADERS_INCLUDE})
 
 if (ENABLE_QT_GUI)
-    set_target_properties(shadps4 PROPERTIES
-#        WIN32_EXECUTABLE ON
-        MACOSX_BUNDLE ON)
-endif()
+    set_target_properties(shadps4 PROPERTIES WIN32_EXECUTABLE ON)
```
Comment (raziel1000) [inline CMakeLists.txt]: "WIN32_EXECUTABLE ON" this one should be disabled to have the console in Qt. You can delete the line I think

Comment (Xphalnos) [reply CMakeLists.txt]: Should I keep the MACOSX_BUNDLE ON ?

**Comment by Xphalnos** [comment]
Fixed.

**Comment by Xphalnos** [comment]
I will reopen it with the correct changes.

**Pull request closed without merging**